screed/tests/test_attriberror.py

```diff
@@ -0,0 +1,82 @@
+import screed
+from screed.DBConstants import fileExtension
+import os
+
+
+class nostring:
+    def __init__(self):
+        self.exists = True
+    def __str__(self):
+        raise AttributeError
+    def __repr__(self):
+        raise AttributeError
+    def __cmp__(self):
+        raise AttributeError
```

Comment:
Why raise the AttributeError here? Isn't that what we're testing in https://github.com/ged-lab/screed/blob/master/screed/screedRecord.py#L118 & elsewhere?